src/lib/services/universal-transparency-scraper.ts

```
@@ -0,0 +1,958 @@
+    for (const method of applicableMethods) {
+      try {
+        const result = await this.runExtractionWithTimeout(method, url, classification)
+        if (result && this.isValidResult(result)) {
+          console.log(`✅ Successfully extracted data using ${method.name}`)
+          return this.normalizeResult(result)
+        }
+      } catch (error) {
+        console.warn(`❌ Method ${method.name} failed:`, error)
+        continue
+      }
+    }
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)